src/app.html

```diff
@@ -5,6 +5,18 @@
 		<title>Bounty Manager</title>
 		<link rel="icon" href="%sveltekit.assets%/favicon.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="title" content="Polkadot Bounty Manager" />
+		<meta
+			name="description"
+			content="Treasury funded UI tool to manage and interact with polkadot bounties. Simplifies and speeds up the curation processes of bounties with one-click actions."
```

Comment:
```suggestion
			content="Treasury funded UI tool to manage and interact with Polkadot bounties. Simplifies and speeds up the curation processes of bounties with one-click actions."
```
Should we respect the branding of Polkadot?